Latest merge causes errors

After pulling the latest version and compiling, I get 24 errors within the Timely crate (commit also caused the CI for timely-dataflow/timely to fail), all errors are either:

error[E0046]: not all trait items implemented, missing: `reborrow`
   --> /Users/name/.cargo/registry/src/index.crates.io-6f17d22bba15001f/timely-0.21.3/src/logging.rs:175:34
    |
175 | #[derive(Serialize, Deserialize, Columnar, Debug, Clone, Hash, Eq, PartialEq, Ord, PartialOrd)]
    |                                  ^^^^^^^^ missing `reborrow` in implementation
    |
    = note: this error originates in the derive macro `Columnar` (in Nightly builds, run with -Z macro-backtrace for more info)

or

error[E0599]: no method named `relax` found for type parameter `CB` in the current scope
  --> /Users/name/.cargo/registry/src/index.crates.io-6f17d22bba15001f/timely-0.21.3/src/dataflow/channels/pushers/exchange.rs:99:38
   |
51 | impl<T: Eq+Data, CB, P, H> Push<Message<T, CB::Container>> for Exchange<T, CB, P, H>
   |                  -- method `relax` not found for this type parameter
...
99 |                 self.builders[index].relax();
   |                                      ^^^^^ method not found in `CB`

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -14,7 +14,7 @@ resolver = "2"
 edition = "2021"
 
 [workspace.dependencies]
-columnar = "0.7"
+columnar = "0.8"
 
 [workspace.lints.clippy]
 type_complexity = "allow"

